package.json

```diff
@@ -9,23 +9,24 @@
   },
   "bugs": "http://github.com/expediagroup/steerage/issues",
   "engines": {
-    "node": ">= 8.0.0"
+    "node": ">= 12.0.0",
+    "npm": ">=6.0.0"
   },
   "dependencies": {
-    "@hapi/hoek": "^8.5.0",
-    "@hapi/joi": "^16.1.7",
+    "@hapi/hoek": "^9.0.4",
     "@hapi/topo": "^3.0.0",
-    "@vrbo/determination": "^3.0.0",
+    "@vrbo/determination": "^3.0.1",
```

Comment:
v4.0.0 of `@vrbo/determination` was recently released so may as well go ahead and bump that in this PR